examples/pylint/w0644_unbalanced_dict_unpacking.py

```diff
@@ -0,0 +1,12 @@
+"""Examples for W0644 unbalanced-dict-unpacking"""
+
+SCORES = {
+    "bob": (1, 1, 3, 2),
+    "joe": (4, 3, 1, 2),
+    "billy": (2, 2, 2, 2),
+}
+
+a, b = SCORES   # unbalanced-dict-unpacking
```

Comment:
Based on your question, I'm not sure if you actually checked whether this line is reported as an error by pylint? You can check this by running PythonTA or pylint on this file.

Please check that and let me know. I'll give you more feedback depending on whether the answer is yes or no.

Reply:
Yes it is reported as an error:
![image](https://github.com/pyta-uoft/pyta/assets/63933334/4f4a22c9-ada6-4673-95b8-f7fe2a4a9889)